js/package.json

```diff
@@ -50,6 +50,7 @@
   },
   "dependencies": {
     "@metaplex/arweave-cost": "^1.0.4",
```

Comment:
anchor is still in this top level package where it's not needed and its still 0.14